src/components/Dialog.js

```diff
@@ -13,16 +13,19 @@ Dialog.propTypes = {
 export default function Dialog({ onDialog, message, nameOfGame }) {
   return (
     <Background onClick={() => onDialog(false)}>
-      <motion.div animate={{ y: 350 }} transition={{ ease: 'easeOut', duration: 1 }}>
-        <DialogBox onClick={(e) => e.stopPropagation()}>
-          <StyledPhrase>{message}</StyledPhrase>
-          <h1>{nameOfGame}</h1>
-          <ButtonContainer style={{ display: 'flex', alignItems: 'center' }}>
-            <PinkButton label="NO" onClick={() => onDialog(false)}></PinkButton>
-            <PrimaryButton label="YES" onClick={() => onDialog(true)}></PrimaryButton>
-          </ButtonContainer>
-        </DialogBox>
-      </motion.div>
+      <DialogBox
+        onClick={(e) => e.stopPropagation()}
+        as={motion.div}
+        animate={{ y: -200, x: -125 }}
+        transition={{ ease: 'easeOut', duration: 1 }}
+      >
+        <StyledPhrase>{message}</StyledPhrase>
+        <h1>{nameOfGame}</h1>
+        <ButtonContainer style={{ display: 'flex', alignItems: 'center' }}>
+          <PinkButton label="NO" onClick={() => onDialog(false)}></PinkButton>
+          <PrimaryButton label="YES" onClick={() => onDialog(true)}></PrimaryButton>
```

Comment:
I don't think the `label` is sufficient here for screen reader user. I would recommend adding a `sr-only` class and adding it to the text between the opening and closing text:
```suggestion
          <PinkButton onClick={() => onDialog(false)}>
            <span className="sr-only">NO</span>
          </PinkButton>
          <PrimaryButton onClick={() => onDialog(true)}>
            <span className="sr-only">YES</span>
          </PrimaryButton>
```

EDIT: never mind, I saw that `label` is a prop of `DefaultButton`. I would probably not call it `label` though, but use `children` instead:
```suggestion
          <PinkButton onClick={() => onDialog(false)}>NO</PinkButton>
          <PrimaryButton onClick={() => onDialog(true)}>YES</PrimaryButton>
```
And then in `DefaultButton`:
```js
export default function DefaultButton({ ...props, children }) {
  return (
    <>
      <Button type="button" {...props}>
        {children}
      </Button>
    </>
  );
}
```